onchain/src/social/interfaceID.cairo

```diff
@@ -0,0 +1,39 @@
+// Generate interface using src5.rs
```

Comment:
This file should be named interface.cairo and contain only ISocialAccount which should contain just two methods: get_public_key, handle_transfer.

Reply:
`is_supported_interface` is a part of isrc5 interface.

Reply:
I've made updates.